settings.gradle.kts

```diff
@@ -127,3 +132,29 @@ dependencyResolutionManagement {
         mavenLocal()
     }
 }
+
+// Develocity
+val isCI = System.getenv("CI") != null // adjust to your CI provider
```

Comment:
the `CI` environment variable is not set anywhere, so it will always be false.
What is this `isCI` variable meant for?

Reply:
Hi @ndr-brt, the `CI` env var is set by [default on Github Actions](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables). It is used to disable background scan publishing on CI (as it can happen that the agents get killed before the publishing is done), as well as enable remote build cache writing on CI.